crates/engine/tree/src/tree/config.rs

```diff
@@ -65,7 +65,7 @@ impl Default for TreeConfig {
             use_state_root_task: false,
             always_compare_trie_updates: false,
             use_caching_and_prewarming: false,
-            cross_block_cache_size: DEFAULT_CROSS_BLOCK_CACHE_SIZE,
+            cross_block_cache_size: DEFAULT_CROSS_BLOCK_CACHE_SIZE_MB,
```

Comment:
hmm, not sure if we should mix this here, I lean towards handling this here in bytes as well as `build_caches`

wdyt @Rjected

Reply:
yep better to be consistent internally, i¡ve pushed changes to manage bytes from `TreeConfig`, done the change MB -> bytes in the cache size setter invocation, ptal